core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -357,6 +357,8 @@ default ClientRequestContext newDerivedContext(RequestId id,
      * Creates a new {@link ClientRequestContext} whose properties and {@link Attribute}s are copied from this
      * {@link ClientRequestContext}, except having different {@link Request}, {@link Endpoint} and its own
      * {@link RequestLog}.
+     *
+     * <p>Note that if you want to copy the properties of the {@link RequestLog}, you should do it by yourself.
```

Comment:
How about simplifying like the following?

    Note that this method does not copy the {@link RequestLog} properties to the derived
    context.

Reply:
Fixed. 😉